### PR TITLE
[REDCAP] adds redcap-project-info and optimises the redcap-import-file

### PR DESCRIPTION
**Pull request by karliwalti**
get fields such as get fields such as project_id, project_title, creation_time, production_time, in_production, project_language, purpose, purpose_other, project_notes, custom_record_label, secondary_unique_field, is_longitudinal, has_repeating_instruments_or_events, surveys_enabled, scheduling_enabled, record_autonumbering_enabled, randomization_enabled, ddp_enabled, project_irb_number, project_grant_number, project_pi_firstname, project_pi_lastname, display_today_now_button, missing_data_codes, external_modules
;; requires to run redcap v >10.0 on host to function properly

### CODE REVIEW

**Comment by karliwalti** [comment]
this is the version requested in #313 . consequently that request can be ignored. 

As we were unable to reproduce the issue with the parsing of redcap data, it now mentions the minimal requirement for redcap

**Comment by mgorges** [comment]
Did you add the piece, which @ddunsmuir asked for 'Please update that procedure to check for both possible string ends or accept an optional flag to deal with this.'? - The rest seems fine to be merged.

**Comment by karliwalti** [comment]
No, as mentioned, I would not be able to test it.

**Comment by mgorges** [comment]
@ddunsmuir can you take a look before we merge this? Thanks!

**Comment by ddunsmuir** [comment]
Conveniently, our REDCap is being updated to LTS version 10.0 on Tuesday. I will test this after that and then merge.

File: modules/redcap/redcap.scm [hunk 1]
```diff
@@ -445,6 +445,56 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
...
+            (begin
+              (httpsclient-close)
+              (let ((output (cadr (redcap:split-headerbody (redcap:data->string)))))
+                 (if (string=? format "json")
+                   ;; If format is json, turn into a list, otherwise just return output
+                   (let ((datalist (redcap:jsonstr->list output)))
+                     (cond
+                       ((not (list? datalist))
+                         ;; If no list returned, json not properly formatted
+                         (log-error "REDCap error: Incomplete json" output)
```
Comment (ddunsmuir) [inline modules/redcap/redcap.scm:473]: This is where I requested a space.

**Pull request merged**